sdk/core/System.ClientModel/src/ModelReaderWriter/IPersistableModel.cs

```diff
@@ -25,7 +25,7 @@ public interface IPersistableModel<out T>
     /// <param name="options">The <see cref="ModelReaderWriterOptions"/> to use.</param>
     /// <returns>A <typeparamref name="T"/> representation of the data.</returns>
     /// <exception cref="FormatException">If the model does not support the requested <see cref="ModelReaderWriterOptions.Format"/>.</exception>
-    T Create(BinaryData data, ModelReaderWriterOptions options);
+    T? Create(BinaryData data, ModelReaderWriterOptions options);
```

Comment:
We should update the changelog to document this as a source breaking change. We should clarify that this will only effect consumers that have nullable references enabled. We should document the fix for when they encounter build breaks.